Sources/AutorestSwift/main.swift

```diff
@@ -7,7 +7,11 @@ do {
     let yamlString = try String(contentsOf: url)
     let decoder = YAMLDecoder()
     let model = try decoder.decode(CodeModel.self, from: yamlString)
-    print(model)
+    let encoder = JSONEncoder()
+    encoder.outputFormatting = .prettyPrinted
+    let jsonData = try encoder.encode(model)
+
+    print(String(data: jsonData, encoding: .utf8)!)
```

Comment:
```suggestion
    if let jsonString = String(data: jsonData, encoding: .utf8) {
      print(jsonString)
    }
```

We haven't enabled swiftlint or other linters on this repo yet, but when we do they will complain hard about a forced unwrap (!).